administrator/language/de-DE/com_plugins.ini

```diff
@@ -38,11 +30,17 @@ COM_PLUGINS_NO_ITEM_SELECTED="Kein Plugin ausgewählt"
 COM_PLUGINS_OPTION_ELEMENT="- Plugindatei wählen -"
 COM_PLUGINS_OPTION_FOLDER="- Typ wählen -"
 COM_PLUGINS_PLUGIN="Plugin"
-COM_PLUGINS_PLUGINS="Plugins"
+COM_PLUGINS_N_QUICKICON="Plugins"
+COM_PLUGINS_N_QUICKICON_1="Plugin"
+COM_PLUGINS_N_QUICKICON_SRONLY="Plugins: %s Plugins sind aktiviert."
+COM_PLUGINS_N_QUICKICON_SRONLY_1="Plugins: Ein Plugin ist aktiviert."
 COM_PLUGINS_SAVE_SUCCESS="Das Plugin wurde gespeichert!"
 COM_PLUGINS_SEARCH_IN_TITLE="Im Titel suchen. Als Präfix „ID:“ verwenden, um nach einer Plugin-IDs zu suchen."
+COM_PLUGINS_TABLE_CAPTION="Tabelle der Plugins"
 COM_PLUGINS_XML_DESCRIPTION="Diese Komponente verwaltet Joomla!-Plugins"
 COM_PLUGINS_XML_ERR="Plugin XML-Datei nicht verfügbar"
 JLIB_HTML_PUBLISH_ITEM="Plugin aktivieren"
 JLIB_HTML_UNPUBLISH_ITEM="Plugin deaktivieren"
-JLIB_RULES_SETTING_NOTES="Vorgenommene Änderungen wirken sich nur auf diese Komponente aus.<br /><em><strong>Vererbt</strong></em>: Eine globale Konfigurationseinstellung oder eine höhere Gruppeneinstellung wird angewendet.<br /><em><strong>Verweigert</strong></em>: Wird immer angewendet, egal was als globale Konfigurationseinstellung oder in einer höheren Gruppeneinstellung definiert wurde. Vererbt sich auf alle Unterelemente.<br /><em><strong>Erlaubt</strong></em>: Aktiviert die Aktion für diese Komponente, sofern es nicht durch eine globale Konfigurationseinstellung überschrieben wird.<br />Der Eintrag muss gespeichert werden, damit sich die errechneten Einstellungen aktualisieren können."
+
+; Alternate language strings for the rules form field
+JLIB_RULES_SETTING_NOTES_COM_PLUGINS="Vorgenommene Änderungen wirken sich nur auf diese Komponente aus.<br /><em><strong>Vererbt</strong></em>: Eine globale Konfigurationseinstellung oder eine höhere Gruppeneinstellung wird angewendet.<br /><em><strong>Verweigert</strong></em>: Wird immer angewendet, egal was als globale Konfigurationseinstellung oder in einer höheren Gruppeneinstellung definiert wurde. Vererbt sich auf alle Unterelemente.<br /><em><strong>Erlaubt</strong></em>: Aktiviert die Aktion für diese Komponente, sofern es nicht durch eine globale Konfigurationseinstellung überschrieben wird.<br />Der Eintrag muss gespeichert werden, damit sich die errechneten Einstellungen aktualisieren können."
```

Comment:
Also ziehen wir diese Sprachstrings nur in 4.x gerade wir hatten dazu ja einen Issue hier? https://github.com/joomlagerman/joomla/issues/794

Reply:
@zero-24 Um da einfach den Projektweg zu gehen bitte nachziehen.
Denke dann ist es auch für Dritte einfacher zu verstehen.

Reply:
Ok dann machen wir das nach und nach in 4.c

Reply:
@zero-24 also in 3.10 dann erstmal weglassen?

Reply:
ja außer du hast lust das nach zu ziehen. Beide Strings funktionieren und beide Strings haben den selben Inhalt.